userguide/tasks.rst

```diff
@@ -365,6 +365,9 @@ Cron jobs are shown in :guilabel:`View Cron Jobs`. Highlight a cron
 job entry to display buttons to :guilabel:`Edit`, :guilabel:`Delete`,
 or :guilabel:`Run Now`.
 
+Manually executing a cron task sends an email to the user specified
+if email has been configured on the %brand% system.
```

Comment:
Might be work a ref to the email config docs or a menuselection to where the user can configure email.